vignettes/pr_review_guidance.Rmd

```diff
@@ -50,6 +50,42 @@ For a pull request to be merged it needs to pass the automated `R CMD check` and
 - all files affected by the implemented changes, e.g. vignettes and templates, are updated
 
 
+# So much Red Tape!
+
+The `{admiral}` development team is aware and sympathetic to the great many checks, processes and
+documents needed to work through in order to do a compliant Pull Request. The `task-list-completed` GitHub
+workflow was created to help reduce this burden on contributors by providing a standardized checklist that compiles information from the Pull Request Review Guidance, [Programming Strategy](programming_strategy.html) and [Development Process](development_process.html) vignettes. 
+
+The next three sections give a high-level overview of what a contributor faces in opening a PR, and how a contributor interacts with the `task-list-completed` workflow in their PR.
+
+## Open a Pull Request
+
+When a contributor opens a Pull Request a lengthy standard text will be inserted into the comment section.  Please do not alter any of the automated text. You are free to add in additional textual information, screenshots, etc at the top of the automated text if needed to clarify or contribute to the discussion around your Pull Request. 
+
+
+```{r echo=FALSE, out.width='120%'}
+knitr::include_graphics('./pr_review_checklist.png')
+```
+
+## Create a Pull Request
+
+After you click the green `Create pull request` button the automated text that was inserted will be turned into a checklist in your Pull Request. Each check box has been drawn from the previously mentioned vignettes and presented in the recommended sequence.  These check boxes are meant to be a helpful aid in ensuring that you have created a compliant Pull Request. 
+
+```{r echo=FALSE, out.width='120%'}
+knitr::include_graphics('./pr_review_checkbox.png')
+```
+
+## Complete all checks
+
+The check boxes are linked to the `task-list-completed` workflow.  You need to check off each box in acknowledgement that you have done you due diligence in creating a compliant Pull Request. GitHub will refresh the Pull Request and trigger `task-list-completed` workflow that you have completed the task.  The PR can not be merged into `devel` until the contributor has checked off each of the check box items.  
+
+```{r echo=FALSE, out.width='120%'}
+knitr::include_graphics('./pr_review_actions.png')
+```
+
+Please don't hesitate to reach out to the admiral team on [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) or through the [GitHub Issues](https://github.com/Roche-GSK/admiral/issues) tracker if you think this checklist needs to be amended or further clarity is needed on a check box item. 
```

Comment:
use pharmaverse instead of Roche-GSK in the GitHub link

Reply:
Updated!  Thanks for the catch and for linking the issue - guess I needed to follow the checklist lol